rfc047-release-process.md

```diff
@@ -44,40 +44,51 @@ Additional labels for pre-release and build metadata are available as extensions
 Chef Client specific clarifications:
 
 ```
-Given a version number MAJOR.MINOR.PATCH, increment the:
+Given a version number MAJOR.MINOR.BUILD:
 
-MAJOR version when the release will break existing functionality
-MINOR version when you add any new features
-PATCH version when you are only fixing bugs, typically regressions
+Increment the MAJOR version when the release will break existing functionality
+Increment the MINOR version when you add any new features
+The BUILD version is increased automatically by the build system
 ```
 
-##### Prerelease versions
+#### Continuous Integration / Continuous Delivery (build system)
+
+The BUILD version is numeric and must be increased for each build.
+
+The build system may consume unreleased BUILD versions. For example, a build which is successful but does not pass integration testing may not have its version reused.
+
+It is left to the build system as to when to reset the BUILD version to zero, provided that newer versions will always compare greater than older versions.
 
-Any prerelease must be versioned in the format MAJOR.MINOR.PATCH.PRETYPE.PREVER, where:
+##### Channels
 
-PRETYPE is the description of the prerelease that the Releaser chooses based on the state of the prerelease, e.g. alpha, beta, rc, dev.
-PREVER is an incremented number starting at zero.
+The build system provides at least these repositories of builds, designated as channels:
 
-For example: 12.0.0.dev.0, 12.2.0.alpha.2, 12.3.0.rc.3
+stable: Official releases
+current: Builds that has passed all automated testing
```

Comment:
I do want to be accurate, but I don't want to be overly specific. I could take out "all", i.e. "builds that have passed automated testing" if we believe there's more automated testing to be done. Are you implying that the promotion process from current to stable would trigger more automated testing? I thought it was just a copy/move operation.

Also, I do want to use the word "release" somewhere with stable. We're not ready to approach regular stable builds with little to no ceremony that show up automatically. For now, builds that make it to stable are our releases, and will continue to have ceremony like blog posts.